ipatests/test_integration/util.py

```diff
@@ -17,51 +17,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import time
 import re
 
 from ipaplatform.paths import paths
 from ipalib.constants import DEFAULT_CONFIG
 
```

Comment:
You're right that the plugin itself does not depend on the integration test base class. As for the functions in the utils module, please move them to tasks.